src/libraries/Microsoft.PowerFx.Core/Parser/ParseNamedFormulasAndUDFResult.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerFx.Core.Errors;
+
+namespace Microsoft.PowerFx.Core.Parser
+{
+    internal class ParseNamedFormulasAndUDFResult
```

Comment:
This name is a bit long, even if it's on-the-nose. How about ParseUserDefinitionResult?

Reply:
nit - rename file to match new name